articles/install-python-qdk.md

```diff
@@ -88,22 +88,23 @@ If you would like to use a different editor, the instructions above have you all
 
 Now you are ready to verify your `qsharp` Python package installation by writing and running a simple Q# program.
 
-1. Create a minimal Q# operation by creating a file called `Operation.qs` and adding the following code to it:
+1. Open Visual Studio Code and create a minimal Q# operation by creating a file called `Operation.qs` and adding the following code to it:
```

Comment:
The instructions seem to present themselves as independent of the editor/IDE a user might use, so I think we should restrict mentions of VS Code to the section where we recommend it as a potential IDE.